### PR TITLE
[FEATURE] ANC PostPartum

### PR DESCRIPTION
**Pull request by anvmn**
#352 
#378

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_982

**Comment by anvmn** [comment]
Peek 2022-07-26 20-20

**Comment by anvmn** [comment]
Selection_984

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/elm/Pages/Prenatal/Activity/Utils.elm [hunk 1]
```diff
@@ -315,6 +354,14 @@ activityCompleted currentDate assembled activity =
             (not <| expectActivity currentDate assembled PrenatalImmunisation)
                 || List.all (immunisationTaskCompleted currentDate assembled) immunisationTasks
 
+        Backend.PrenatalActivity.Model.Breastfeeding ->
+            -- @todo
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Activity/Utils.elm]: Anything to write on this todo? or it doesn't matter?

Comment (nedSaf) [reply client/src/elm/Pages/Prenatal/Activity/Utils.elm]: (Here and below)

File: client/src/elm/Pages/Prenatal/Participant/View.elm [hunk 2]
```diff
@@ -224,78 +224,101 @@ viewPrenatalActionsForNurse :
...
+                )
+                lastEncounterType
+                |> Maybe.withDefault
+                    -- When there're no encounters, we allow to
+                    -- create postpartum encounter.
+                    True
+
+        -- Whether first prenatal encounter for person is in process.
+        -- This is True when there's only one encounter, it's active, and
+        -- it is of type NurseEncounter.
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Participant/View.elm]:
```suggestion
        -- Whether the first prenatal encounter for the person is in process.
        -- This is True when there's only one encounter, it's active, and
        -- it is of type NurseEncounter.
```

File: client/src/elm/Pages/Prenatal/Participant/View.elm [hunk 3]
```diff
@@ -224,78 +224,101 @@ viewPrenatalActionsForNurse :
...
+        firstEncounterInProcess =
+            (List.length encounters == 1)
+                && isJust maybeActiveEncounterId
+                && (Maybe.map ((==) NurseEncounter) lastEncounterType |> Maybe.withDefault False)
+
+        -- Whether postpartum encounter for person is in process.
+        -- This is True when current active encounter is of
+        -- NursePostpartumEncounter type.
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Participant/View.elm]:
```suggestion
        -- Whether postpartum encounter for the person is in process.
        -- This is True when the current active encounter is of
        -- NursePostpartumEncounter type.
```

**Review by nedSaf — APPROVED** [review]
👍 Small nitpicks.

File: client/src/elm/Pages/Prenatal/Activity/Utils.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/Prenatal/Activity/Utils.elm]: These are just placehoders for activities that will be implemented in follow up PRs

Comment (nedSaf) [reply client/src/elm/Pages/Prenatal/Activity/Utils.elm]: 👍

**Pull request merged**